web/impact/impact/tests/test_user_history_view.py

```diff
@@ -64,6 +64,11 @@ def test_user_joined_startup(self):
                 id=startup.organization.id)
             self.assertEqual(startup_str, events[0]["description"])
 
+    '''
```

Comment:
Can you please make this a comment, rather than a docstring-in-space?

Reply:
done.